### PR TITLE
NeuralChat README refinement

### PR DESCRIPTION
**Pull request by ftian1**
## Type of Change

documentation change

## Description

This PR is used to refine the neuralchat documentation

### CODE REVIEW

File: neural_chat/README.md [hunk 1]
```diff
@@ -1,214 +1,157 @@
 <div align="center">
 
-Intel® Neural Chat
+Intel® NeuralChat
```
Comment (hshen14) [inline neural_chat/README.md:3]: Remove Intel@ since it's not a product.

File: neural_chat/README.md [hunk 2]
```diff
@@ -1,214 +1,157 @@
...
 ===========================
-<h3> An open-source Python library that empowers you to customize your chatbot with a diverse range of plugins.</h3>
+<h3> An open-source Python library that empowers users to easily create a chatbot with a diverse range of plugins.</h3>
```
Comment (hshen14) [inline neural_chat/README.md:5]: Create your own chatbot on any device within minutes!

File: neural_chat/README.md [hunk 3]
```diff
@@ -1,214 +1,157 @@
...
-**command line experience**
+## Installation
 
-```shell
-neuralchat textchat --query "Tell me about Intel Xeon Scalable Processors."
-```
+NeuralChat is seamlessly integrated into the Intel Extension for Transformers. Please refer to [Installation](../docs/installation.md) page for step by step instructions.
 
-**Python API experience**
+## Get Started
```
Comment (hshen14) [inline neural_chat/README.md:44]: Getting Started

File: neural_chat/README.md [hunk 4]
```diff
@@ -1,214 +1,157 @@
...
-```
-
-
-<a name="quickstartserver"></a>
-# Quick Start Server
-
-Users can have a try of NeuralChat server with [NeuralChat Server Command Line](./server/README.md).
+from neural_chat import build_chatbot, PipelineConf, plugins
 
+plugins.retrival.enabe = True
```
Comment (lvliang-intel) [inline neural_chat/README.md:119]: enabe ====> enable

File: neural_chat/README.md [hunk 5]
```diff
@@ -1,214 +1,157 @@
...
+plugins.retrival.path = '/path/to/user/doc'
+conf = PipelineConf(plugins=plugins)
```
Comment (lvliang-intel) [inline neural_chat/README.md:121]: PipelineConf(plugins=plugins) ====> PipelineConf(plugin=plugins)

Comment (hshen14) [reply neural_chat/README.md:121]: If plugin = plugins, then we should use plugins = plugins

**Pull request merged**